File: Decorato.java/src/com/parttern/decorator/ChocolateMilkTea.java
package com.parttern.decorator;

public class ChocolateMilkTea extends  MilkTea{
    private double price = 1.5;

    @Override
    public String doMilkTea() {
        return "Trà sữa Socola";
    }

    @Override
    public double cost() {
        return this.price;
    }
}

File: Decorator/ChocolateMilkTea.cs
namespace Decorator {
    internal class ChocolateMilkTea : MilkTea {
        private readonly double price = 1.5;

        public override string DoMilkTea() {
            return "Trà sữa Socola";
        }
        public override double Cost() {
            return price;
        }
    }
}
